dart/lib/src/protocol/sentry_transaction.dart

```diff
@@ -70,6 +74,15 @@ class SentryTransaction extends SentryEvent {
     json['start_timestamp'] =
         formatDateAsIso8601WithMillisPrecision(startTimestamp);
 
+    final ms = measurements;
+    if (ms != null && ms.isNotEmpty) {
+      final map = <String, dynamic>{};
+      for (final m in ms) {
+        map[m.name] = m.toJson();
+      }
+      json['measurements'] = map;
+    }
```

Comment:
Can we use a `map` instead? Similar to `SentryEvent#toJson` as well

Reply:
I'm not sure what you are referring to, could you hive me a hint? In `SentryEvent` we have list/array in the JSON, which we do not have here, it's a map of maps (Data Structure).

Reply:
Sure, it's just a nit tho, feel free to ignore it, I guess I meant to use the extension methods like `.map` and `.where` as the other methods do, but indeed, the others are a List as well.